ironfish/src/mining/webhooks/webhookNotifier.ts

```diff
@@ -62,4 +77,12 @@ export abstract class WebhookNotifier {
       `Error while sending payout transaction: ${ErrorUtils.renderError(error, true)}`,
     )
   }
+
+  private renderHashHex(hashHex: string, explorerUrl: string | null): string {
+    if (explorerUrl == null) {
+      return `\`${hashHex}\``
+    }
+
+    return `[${hashHex}](${explorerUrl + hashHex})`
```

Comment:
Does this work in discord? Where does this work?

Reply:
Works with Discord, but not with Lark. I'll change it to use just the URL